docs/source/index.rst

```diff
@@ -109,6 +109,13 @@ In the TestNet.
 
 In the MainNet.
 
+.. rst-class:: column column2
+
+:ref:`Store XNS in a hardware wallet <ledger-nano>`
```

Comment:
How does this read? specifically the "<ledger-nano>`" part? is <ledger-nano> a logo or clickable link? If so...
We would say: Store XNS in your <ledger-nano> hardware wallet

Reply:
"`" is syntax sugar for links.

Reply:
so the <ledger-nano> is not visible to the reader?

Reply:
<ledger-nano> is not visible

Reply:
"ledger-nano" I mean

Reply:
Ah ok. Then this comment can be ignored/deleted